openedx/features/offline_mode/tasks.py

```diff
@@ -37,4 +37,4 @@ def generate_offline_content_for_block(block_id, html_data):
     """
     block_usage_key = UsageKey.from_string(block_id)
     xblock = modulestore().get_item(block_usage_key)
-    generate_offline_content(xblock, html_data)
+    SaveOfflineContentToStorage().generate_offline_content(xblock, html_data)
```

Comment:
Arguments should be moved to init parameters for more convenient use as self-attributes.

Reply:
Fixed